feconf.py

```diff
@@ -348,6 +352,16 @@ def get_empty_ratings():
     },
 }
 
+# The list contains dicts with category of the classifier i.e.
+# 'StringClassifiers' or 'CodeClassifiers' and the IDs of the
+# corresponding classifiers.
+ANSWER_CLASSIFIER_CLASSES = [{
+    'category': 'StringClassifiers',
```

Comment:
You may not need a category here -- if it's here just for parity with interactions, you can remove it. For interactions, the category is needed to arrange them in the modal that pops up when the creator is asked to select an interaction.